flexmailer.php

```diff
@@ -143,13 +143,13 @@ function flexmailer_civicrm_alterSettingsFolders(&$metaDataFolders = NULL) {
  */
 function flexmailer_civicrm_navigationMenu(&$menu) {
   _flexmailer_civix_insert_navigation_menu($menu, 'Administer/CiviMail', [
-    'label' => ts('Flexmailer Settings', array('domain' => 'org.civicrm.flexmailer')),
+    'label' => E::ts('Flexmailer Settings'),
     'name' => 'flexmailer_settings',
     'permission' => 'administer CiviCRM',
-    'child' => array(),
+    'child' => [],
     'operator' => 'AND',
     'separator' => 0,
-    'url' => CRM_Utils_System::url('civicrm/admin/flexmailer', 'reset=1', TRUE),
+    'url' => CRM_Utils_System::url('civicrm/admin/setting/flexmailer', 'reset=1', TRUE),
```

Comment:
Just to verbalize: I think this checks-out. Ordinarily, one might be concerned about changing the path for a page. However, in this case, the item is registered via `hook_civicrm_navigationMenu` which (IIRC) means that the menu should always be up-to-date with the declaration therein.

By contrast, some extensions use the installer to insert a row into `civicrm_navigation` - with that technique, would be more concerned about downstream folks fiddling with the configuration data. But given the hook that's actually used here, it doesn't seem like a concern